plugins/@grouparoo/hubspot/__tests__/integration/destination-mapping-options.ts

```diff
@@ -0,0 +1,94 @@
+import path from "path";
```

Comment:
This is not an integration test. Let's put it in a folder called "export" like the other plugins with these tests.